lib/pltr/v2/reducers/tours.js

```diff
@@ -8,56 +9,64 @@ export default function tours(state = INITIAL_STATE, action) {
     case SET_TOUR_FEATURE:
       const newState = Object.assign({}, INITIAL_STATE)
       newState.feature = action.payload //an object with feature: 'string', featureId: int
+      newState.steps = ACTS_TOUR_STEPS
       newState.showTour = true
       return { ...state, ...newState }
     //main tour callback
     case TOUR_NEXT: {
       let transitionSteps = []
       let endStep
-      state.b2bTransition = false
+      const newState = Object.assign({}, state)
+
+      //ensure the tour runs at the next step
+      newState.loading = false
+      newState.run = true
+
       //switch based on which tour is running -> to determine when the
       //tour callback should delay until next step's component is mounted
-      switch (state.feature.name) {
+      switch (newState.feature.name) {
         case 'acts':
-          transitionSteps = [0, 4, 5, 6]
-          endStep = 9
+          transitionSteps = [0,4,5,6,7]
+          endStep = 8
           break
         default:
-          state.transitioning = false
-      }
-      //delay the next step until component associated with the step is mounted
-      if (transitionSteps.includes(state.stepIndex)) {
-        state.transitioning = true
-        //need to anticipate back-to-back delays
-        if (transitionSteps.includes(state.stepIndex + 1)) {
-          state.b2bTransition = true
-        }
-      } else {
-        state.transitioning = false
+          newState.transitioning = false
```

Comment:
Isn't this redundant because we set transitioning later?

Reply:
Ah so it's necessary here because if the previous step's transitioning was true then transitioning comes in as true and has to be set to false each time, but where it is redundant is in the TOUR_TRANSITION action reducer below. Removing that line now.